vello_encoding/src/resolve.rs

```diff
@@ -422,6 +422,8 @@ impl Resolver {
                             scale = transform.matrix[0];
                             font_size *= scale;
                             transform.matrix = [1.0, 0.0, 0.0, 1.0];
+                            // Hinting assumes that the baseline is rounded; otherwise, we are missing the point of hinting.
+                            transform.translation[1] = transform.translation[1].round();
```

Comment:
The glyph transform is applied later (when emitting the transform stream in resolve) and the translation is also scaled there so I think that’s the proper place to apply rounding.